### DISCUSSION TITLE
Accept contributions and pull req. from collaborators?

### DISCUSSION THREAD

**Discussion by atcarter714** [microsoft/semantic-kernel#549] [General]
I saw my guy @xoofx was working on something that sounded strange/interesting and came across this! :D

I started reading about it and became very interested, because I have a new org and we are working on a really big, comprehensive ecosystem and projects within it. This project sounds absolutely fantastic, and also kinda seems right up my alley ...

So I was curious if this is something Arkaen and I could get involved in and get behind, contribute to, etc. Or if this is not possible for some reason. I have to consider a lot of stuff before I dive in and tell people start using things :D

If so, I'd like to talk to someone about it, take some time to study/analyze and use it and potentially be a contributor.

Regards,

Aaron

**Comment by dluc** [comment]
hi Aaron, we accept external contributions, some notes here https://github.com/microsoft/semantic-kernel/blob/main/CONTRIBUTING.md plus the usual Contributor License Agreement the are part of the pull request flow. For minor contributions the process is the typical GitHub flow. For major changes we encourage proposals and design conversations before sending PRs, e.g. using this Discussions board.
